docfx_project/ziti/overview.md

```diff
@@ -2,6 +2,15 @@
 uid: zitiOverview
 ---
 
+# Why?
+
```

Comment:
feels like it needs a setup sentence.  Here's how this appears when rendered.  Feels like it needs a setup before just bullet points. Also bullet one feels marketing-ish. We should focus on the developer not the manager/marketer. Maybe just "You don't want to worry about security and networking. OpenZiti has you covered."

![image](https://user-images.githubusercontent.com/46322585/158039249-ffc601be-0b1b-4576-9ba7-d84e31e4d8ca.png)

Reply:
- Maybe go with 'Why did we create Ziti'?
- For the bullets, each should be independent of Ziti, it's the why we created it in the first place. Thus maybe update to 'You want to focus on writing code, not worrying about security and networking' or 'You don't want to worry about security and network, you just want to focus on writing code'
- Finally, I was actually hoping it would go to this page - https://openziti.github.io/ ... I could see why its valuable on both pages.